documentation/course/e2b/README.md

```diff
@@ -0,0 +1,215 @@
+| 19. Describe what the application generally does, the major components of the application and the major data flows.                                                                                               |            |
+| 20. Describe the instrumentation included in the application that allows for the health and performance of the application to be monitored.                                                                       |            |
+| 21. Describe the business justification for the system.                                                                                                                                                           |            |
+| 22. Describe the rationale for picking the system development language over other options in terms of initial development cost versus long term maintenance cost.                                                 |            |
+| 23. Describe the systems analysis process that was used to come up with the system architecture and product selection phase of the system architecture.                                                           |            |
+| 24. Who besides the original customer might have a use for or benefit from using this system?                                                                                                                     |            |
+| 25. What percentage of the users use the system in browse mode versus update mode?                                                                                                                                |            |
+| 26. What is the typical length of requests that are transactional?                                                                                                                                                |            |
+| 27. Do you need guaranteed data delivery or update, or the system tolerate failure?                                                                                                                               |            |
+| 28. What are the up-time requirements of the system?                                                                                                                                                              |            |
+| 29. Describe where the system architecture adheres or does not adhere to standards.                                                                                                                               |            |
+| 30. Describe the project planning and analysis approach used on the project.                                                                                                                                      |            |
+| 1. Describe the Client/Server application architecture.                                                                                                                                                           |            |
```

Comment:
I did finish primary school and I think that 1 is smaller than 30 thus should be above it.

Jokes aside, maybe we should enumerate them in a different way? Starting counting in the middle from 1 seems kinda weird, even though I know it's a different category of questions.